tst/EnergyPlus/unit/DesiccantDehumidifiers.unit.cc

```diff
@@ -2898,7 +2898,7 @@ TEST_F(EnergyPlusFixture, DesiccantDehum_OnOASystemTest)
     auto &finalSysSizing = state->dataSize->FinalSysSizing(1);
     auto &sysSizPeakDDNum = state->dataSize->SysSizPeakDDNum(1);
     EXPECT_ENUM_EQ(finalSysSizing.coolingPeakLoad, DataSizing::PeakLoad::SensibleCooling);
-    EXPECT_EQ(finalSysSizing.SizingOption, DataSizing::NonCoincident);
+    EXPECT_ENUM_EQ(finalSysSizing.SizingOption, DataSizing::SizingConcurrence::NonCoincident);
```

Comment:
At this point, all unit test changes are simply updates to work with other codes changes (new enums, new function names, etc.).